features/apimgt/org.wso2.carbon.apimgt.core.feature/resources/dbscripts/mssql.sql

```diff
@@ -303,6 +303,16 @@ CREATE TABLE AM_API_RESOURCES (
   FOREIGN KEY (RESOURCE_CATEGORY_ID) REFERENCES AM_RESOURCE_CATEGORIES(RESOURCE_CATEGORY_ID)
 );
 
+CREATE TABLE AM_API_ADDITIONAL_PROPERTIES (
+  UUID VARCHAR(255),
+  API_ID VARCHAR(255),
+  PROPERTY_KEY VARCHAR(255),
+  PROPERTY_VALUE VARCHAR(255),
+  UNIQUE (UUID),
+  PRIMARY KEY (API_ID, PROPERTY_KEY),
```

Comment:
Any reason API_ID, PROPERTY_KEY being PRIMARY KEY? These should be unique constraints. UUID seems to be the PRIMARY_KEY for this table

Reply:
fixed